api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java

```diff
@@ -231,6 +231,10 @@ private void loadAppsAndExtensions() throws UnsupportedEncodingException {
             enableCheckIn(config);
         }
 
+        if (config.isComponentEnabled(Components.MCOE_TRIAGE)) {
```

Comment:
I'm wondering if we want to name this something like "maternal triage" or "mch triage"?

Reply:
Thanks!